rsocket/ResumeManager.h

```diff
@@ -73,10 +75,43 @@ class ResumeManager {
   // remote side.
   virtual ResumePosition impliedPosition() const = 0;
 
-  // This gets called when a new stream is opened.
-  virtual void onStreamOpen(StreamId streamId, FrameType frameType) = 0;
-
   // This gets called when a stream is closed.
   virtual void onStreamClosed(StreamId streamId) = 0;
+
+  // Return the StreamIds of locally originating REQUEST_STREAMs
+  virtual std::unordered_set<StreamId> getRequesterRequestStreamIds() = 0;
```

Comment:
I would prefer if this returns `(StreamId -> { Allowance, Token })` or something instead of requiring 3 calls per stream.